test/e2e/legacyantreapolicy_test.go

```diff
@@ -633,6 +633,8 @@ func testLegacyACNPPriorityOverrideDefaultDeny(t *testing.T) {
 func testLegacyACNPAllowNoDefaultIsolation(t *testing.T, protocol v1.Protocol) {
 	if protocol == v1.ProtocolSCTP {
 		skipIfProviderIs(t, "kind", "OVS userspace conntrack does not have the SCTP support for now.")
+		// ESX doesn't support IPv6 SCTP communication between different ESX hosts for now.
```

Comment:
this is not enough context. You have to specify that this is a reference to our IPv6 testbeds.

> SCTP testing is failing on our IPv6 CI testbeds at the moment. This seems to be related to an issue with ESX networking for SCTPv6 traffic when the Pods are on different Node VMs which are themselves on different ESX hosts. We are investigating the issue and disabling the tests for IPv6 clusters in the meantime.

same below

Reply:
Done.